libs/platform/experience/src/mixins/content.mixin.ts

```diff
@@ -74,10 +74,10 @@ export const ContentMixin = <
     @asyncState()
     protected componentContent = valueType(this.content$);
 
-    protected $options = signal(this.contentController.getOptions(), {
```

Comment:
why do you need to make it public?

Reply:
The font controller needs it

Reply:
Too big change to get such small effort. Then maybe it is better not to use the controller? mixin for example. Or another way - create a public method inside component that will return options, or specific option and controller will use it